tests/unit/parsers/test.manifestjson.js

```diff
@@ -307,6 +307,73 @@ describe('ManifestJSONParser', () => {
     expect(manifestJSONParser.isValid).toEqual(true);
   });
 
+  it('should warn when gecko.strict_min_version is set below 113.0 and gecko_android is present', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const json = validManifestJSON({
+      browser_specific_settings: {
+        gecko: {
+          strict_min_version: '100.0',
+        },
+        gecko_android: {},
+      },
+    });
+
+    const manifestJSONParser = new ManifestJSONParser(
+      json,
+      addonLinter.collector
+    );
+
+    const { warnings } = addonLinter.collector;
+    expect(warnings).toEqual([
+      // Slightly awkward, but gecko_android is in the compatibility data as
+      // a Firefox key too, so the message is duplicated for each app, even
+      // though it's only really an Android property.
```

Comment:
We probably want to fix this, since this also makes a warning appear in the following scenario:
```json
"browser_specific_settings": {
  "gecko": {
    "strict_min_version": "57.0"
  },
  "gecko_android": {
    "strict_min_version": "113.0"
  }
}

But I think this should be fixed in the compatibility data if possible ? Can we make `gecko_android` completely Android specific ?

Reply:
@willdurand thoughts on that ?

Reply:
I suppose we could mark it as Android-only.

Reply:
https://github.com/mdn/browser-compat-data/pull/20810